develop/addons/schema-driven-forms/customising-form-behaviour/actions.rst

```diff
@@ -87,7 +87,7 @@ Updating button properties
 As with regular widgets, it is sometimes useful to set properties on
 buttons after they have been instantiated by *z3c.form*. One common
 requirement is to add a CSS class to the button. The standard edit form
-in*plone.directives.form* does this, for example, to add Plone’s
+in *plone.directives.form* does this, for example, to add Plone’s
```

Comment:
@staeff would you mind to check ? If we can update this with plone.dexterity we can include that into our next release, which would be super awesome ! :)

Thanks  a lot !